components/user/EventRow.module.css

```diff
@@ -47,3 +47,15 @@
   opacity: 0 !important;
   display: none !important;
 }
+
+.truncatedHash {
+  direction: rtl;
```

Comment:
This is a [tricky trick](https://davidwalsh.name/css-ellipsis-left)

Reply:
oh that's pretty cool, nice!